generators/2.0/java/templates/build.gradle.ejs

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java-library'
+    id 'application'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+repositories {
+    mavenLocal()
```

Comment:
maybe something to remove later (at it would be only useful for us to test local versions)

Reply:
Removed 👍